it/it-tests/src/test/java/pageobjects/projects/FacetItem.java

```diff
@@ -38,4 +38,14 @@ public FacetItem shouldHaveValue(String key, String value) {
   public void selectValue(String key) {
     this.elt.$(".facet[data-key=\"" + key + "\"]").click();
   }
+
+  public FacetItem selectOptionItem(String value) {
+    this.elt.$(".Select-input input").val(value).pressEnter();
+    return this;
+  }
+
+  public FacetItem sortList(Integer idx) {
```

Comment:
Ok I'll change it, but I don't think I can rely on that method then. Because not every facets have the left button that does a descending sort and the right button an ascending sort.